api/pkg/service/status/status_http_test.go

```diff
@@ -15,24 +15,54 @@
 package status
 
 import (
-	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"testing"
 
 	"github.com/ikawaha/goahttpcheck"
 	"github.com/stretchr/testify/assert"
+	"gotest.tools/golden"
+
 	"github.com/tektoncd/hub/api/gen/http/status/server"
 	"github.com/tektoncd/hub/api/gen/status"
+	"github.com/tektoncd/hub/api/pkg/testutils"
 )
 
 func TestOk_http(t *testing.T) {
+	tc := testutils.Setup(t)
+	testutils.LoadFixtures(t, tc.FixturePath())
+
+	checker := goahttpcheck.New()
+	checker.Mount(
+		server.NewStatusHandler,
+		server.MountStatusHandler,
+		status.NewStatusEndpoint(New(tc)),
+	)
+
+	checker.Test(t, http.MethodGet, "/").Check().
+		HasStatus(http.StatusOK).Cb(func(r *http.Response) {
+		b, readErr := ioutil.ReadAll(r.Body)
+		assert.NoError(t, readErr)
+		defer r.Body.Close()
+
+		res, err := testutils.FormatJSON(b)
+		assert.NoError(t, err)
+
+		golden.Assert(t, res, fmt.Sprintf("%s.golden", t.Name()))
+	})
+}
+
+func TestNotOk_http(t *testing.T) {
+	tc := testutils.Setup(t)
+
+	tc.DB().Close()
```

Comment:
Do you know if the order of execution for the test is always the same? :thinking:
all the test in the package uses the same DB connection if `TestNotOk_http` executed first then the other will fail

Reply:
can you check if the order of execution is always the same ?

Reply:
I don't think we should make assumptions about the order of tests.

Reply:
then I think `tc.DB().Close()` is not a good idea because we use one db connection for all tests in a package ...wdyt?